### PR TITLE
Fixed bug while updating registrar 

### PR DESCRIPTION
**Pull request by soumita-dsi**

### CODE REVIEW

File: packages/client/src/views/SysAdmin/Team/user/userCreation/UserReviewForm.tsx [hunk 1]
```diff
@@ -249,6 +249,7 @@ const mapDispatchToProps = (dispatch: Dispatch, props: IFullProps) => {
 
       if (variables.user.signature) {
         delete variables.user.signature.name
+        delete variables.user.signature.__typename //to fix updating registrar bug
```
Comment (rikukissa) [inline packages/client/src/views/SysAdmin/Team/user/userCreation/UserReviewForm.tsx:252]: Where is this field coming from? If we need to remove it here, do we need to fetch it from the backend in the first place?

Comment (soumita-dsi) [reply packages/client/src/views/SysAdmin/Team/user/userCreation/UserReviewForm.tsx:252]: Thanks Riku. I spent much time today on thinking about it. This __typename field is automatically added from graphql when user is fetched. So I couldn't find out any easier way to remove it from backend and went for this aproach.

Comment (soumita-dsi) [reply packages/client/src/views/SysAdmin/Team/user/userCreation/UserReviewForm.tsx:252]: when did you console log it? while updating registrar or creating registrar? __typename exists only at the time of updating registrar.

Comment (soumita-dsi) [reply packages/client/src/views/SysAdmin/Team/user/userCreation/UserReviewForm.tsx:252]: image

Comment (soumita-dsi) [reply packages/client/src/views/SysAdmin/Team/user/userCreation/UserReviewForm.tsx:252]: I got this in signature while updating registrar

Comment (rikukissa) [reply packages/client/src/views/SysAdmin/Team/user/userCreation/UserReviewForm.tsx:252]: Thanks! Yea, it was just me trying to see it when creating the user. I'm just now thinking if the view is the best place to get rid of those extra fields. If we don't have a universal way of removing all __typenames from query results, then maybe it is... Ok, I'm happy with this 👍

**Review by rikukissa — APPROVED** [review]

**Pull request merged**